sadu-queries2/src/main/java/de/chojo/sadu/queries/calls/ResultCall.java

```diff
@@ -0,0 +1,15 @@
+/*
+ *     SPDX-License-Identifier: LGPL-3.0-or-later
+ *
+ *     Copyright (C) RainbowDashLabs and Contributor
+ */
+
+package de.chojo.sadu.queries.calls;
+
+import de.chojo.sadu.mapper.util.Results;
+import de.chojo.sadu.queries.call.Call;
+
+public class ResultCall extends Call {
+    public ResultCall(Results results) {
```

Comment:
Is the result accessible ?

Reply:
I dont even know why this class exists. Seems not to be in use anyway.